src/steps/sql-admin/index.test.ts

```diff
@@ -153,6 +154,10 @@ describe('#fetchSQLInstances', () => {
           automatedBackupsEnabled: { type: 'boolean' },
           kmsKeyName: { type: 'string' },
           connectionName: { type: 'string' },
+          externalScriptsEnabled: { type: 'string' }, // should we convert this to boolean?
+          userConnections: { type: 'number' },
+          remoteAccess: { type: 'string' }, // should we convert this to boolean?
+          traceFlag: { type: 'string' },
```

Comment:
Good question! Because the wording is "on"/"off" instead of "true"/"false", we probably want to keep these as they are. If the values were instead "true"/"false" (but strings) we'd want to convert them to booleans most likely because we could avoid the issue of "true" !== "TRUE" !== "True".